src/main/java/hudson/plugins/analysis/collector/AnalysisGraphConfiguration.java

```diff
@@ -77,7 +77,7 @@ public boolean getCanDeacticateOtherTrendGraphs() {
 
     @Override
     public String serializeToString() {
-        return super.serializeToString() + SEPARATOR + serializeBoolean(canDeactivateGraphs);
+        return super.serializeToString() + serializeBoolean(canDeactivateGraphs);
```

Comment:
For the future, it would be better to do such changes in a separate upstream PR, to keep the diffs shorter and clearer. (So long as the upstream PR is unmerged the downstream will show the aggregate diff, unless you use a special `compare` URL, but the point is that you can probably ask for and get the upstream PR merged quickly since it should not be controversial or require detailed review. Once it is merged, then `git merge master` from the downstream PR and push, and the downstream PR’s diff cleans up automatically.)